tdrs-backend/tdpservice/search_indexes/models/tribal.py

```diff
@@ -1 +1,190 @@
 """Models representing parsed TRIBAL data file records submitted to TDP."""
+
+import uuid
+from django.db import models
+from django.contrib.contenttypes.fields import GenericRelation
+from tdpservice.data_files.models import DataFile
+from tdpservice.parsers.models import ParserError
+
+
+class Tribal_TANF_T1(models.Model):
+    """
+    Parsed record representing a T1 data submission.
+
+    Mapped to an elastic search index.
+    """
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    datafile = models.ForeignKey(
+        DataFile,
+        blank=True,
+        help_text='The parent file from which this record was created.',
+        null=True,
+        on_delete=models.CASCADE,
+        related_name='tribal_t1_parent'
+    )
+
+    error = GenericRelation(ParserError)
+    RecordType = models.CharField(max_length=156, null=True, blank=False)
+    RPT_MONTH_YEAR = models.IntegerField(null=True, blank=False)
+    CASE_NUMBER = models.CharField(max_length=11, null=True, blank=False)
+    FIPS_CODE = models.CharField(max_length=2, null=True, blank=False)
```

Comment:
@elipe17 this should be tribe_code, and its stored in the header record, as shown below:
`HEADER20214A00142TAN1EU`

tribe code in the header above is `142`